spec/index.htm

```diff
@@ -68,77 +68,28 @@
     <h1>|Arguments|</h1>
 
     <emu-production name="Arguments" params="Yield">
-      <emu-rhs>()</emu-rhs>
+      <emu-rhs>( )</emu-rhs>
       <emu-rhs>(<emu-nt params="?Yield">ArgumentList</emu-nt>)</emu-rhs>
       <ins class="block">
-        <emu-rhs>(<emu-nt params="?Yield">ArgumentList</emu-nt>,)</emu-rhs>
-        <emu-rhs a="Arguments--ArgumentList-AssignmentExpression">(<emu-nt params="?Yield">ArgumentList</emu-nt>, ...<emu-nt params="In, ?Yield">AssignmentExpression</emu-nt>)</emu-rhs>
```

Comment:
Oh nvm, these are spreads not rests -- so trailing comma is fine (per offline convo).